docker/tests/01/goss.yaml

```diff
@@ -2,8 +2,8 @@ file:
   /opt/web3signer/bin/web3signer:
     exists: true
     mode: "0755"
-    owner: root
-    group: root
+    owner: web3signer
+    group: web3signer
```

Comment:
Bear in mind that changing from the default user being root to web3signer introduces backwards incompatibility when directories are mounted into the docker container as ownership rights won't match up.  You'll want to at least note that in the changelog and may need to give some advance warning about the change.

Reply:
that is a good point, I have made a changelog entry.